metricbeat/docs/fields.asciidoc

```diff
@@ -43262,7 +43262,7 @@ format: percent
 *`kubernetes.container.cpu.usage.limit.pct`*::
 +
 --
-CPU usage as a percentage of the defined limit for the container (or total node allocatable CPU if unlimited)
+CPU usage as a percentage of the defined limit for the container (or total node allocatable CPU if unlimited). If the container CPU limits are missing and the `node` and `state_node` metricsets are both disabled or not configured on that node, this metric will be missing entirely.
```

Comment:
what do you mean by `or not configured`?

Reply:
`not configured` = `not in the configuration`. any better idea how to convey that?

Reply:
in this case ```the `node` and `state_node` metricsets are both disabled``` = `not configured` ? maybe then just leave first part?